rust/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[workspace]
+members = ["osm2lanes", "osm2lanes-web"]
+
+[profile.release]
+# less code to include into binary
```

Comment:
I'm curious, are these best practices for yew projects? Performance is definitely not a concern at all for something like this project. I'm fine with all of these, just curious about the origin. FWIW I usually default to

```
[profile.dev.package."*"]
opt-level = 3
```

so even when you compile in debug mode, all the external dependencies are in release mode